include/cadet/cadet.h

```diff
@@ -1,7 +1,7 @@
 // =============================================================================
 //  CADET - The Chromatography Analysis and Design Toolkit
 //  
-//  Copyright © 2008-2024: The CADET Authors
```

Comment:
Or was this supposed to be the only source file where its still included? I dont see why.

Reply:
no, I guess this was a mistake. I used `sed` to edit all files maybe there was some hickup.